ElrondNetwork/api.elrond.com
# update swagger docs

## Type

## Problem setting
-  refactor endpoint responses
  
## Proposed Changes
- Use specific response decorators
- Update swagger.md doc
- Add search tag in swagger

## How to test
- run npm run start:mainnet and verify response body on swagger for every endpoint
